cypress/e2e/po/pages/diagnostics.po.ts

```diff
@@ -0,0 +1,30 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+
+/**
+ * Diagnostics page
+ */
+export default class DiagnosticsPagePo extends PagePo {
+  static url = '/diagnostic'
+
+  constructor(pageUrl = DiagnosticsPagePo.url) {
+    DiagnosticsPagePo.url = pageUrl;
+    super(DiagnosticsPagePo.url);
+  }
+
+  downloadDiagnosticsPackageClick() {
+    const btn = new AsyncButtonPo('[data-testid="diagnostics-download-diagnostic-package"]');
```

Comment:
niggly. these two functions should just return the button. the test itself can then click. this makes the tests easier to understand

```
diagnosticsPage.diagnosticPackageButton().click()
```